lib/rules/no-nested-tests.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @fileoverview Forbid usage of nested QUnit.test()
+ * @author Aliaksandr Yermalayeu
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const utils = require("../utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "Forbid nested QUnit.test() calls",
+            category: "Possible Errors"
+        },
+        messages: {
+            noNestedTests: "Usage of QUnit.test inside of another QUnit.test is not allowed."
```

Comment:
This message is fine, but I think it could be slightly simplified, which could benefit people who don't have English as their native language. What do you think?

```suggestion
            noNestedTests: "Using QUnit.test inside of another QUnit.test is not allowed."
```